.gitignore

```diff
@@ -7,6 +7,7 @@
 /unity-environment/[Uu]nity[Pp]ackage[Mm]anager/
 /unity-environment/Assets/AssetStoreTools*
 /unity-environment/Assets/Plugins*
+/unity-environment/Assets/Gizmos*
```

Comment:
What are Gizmos ? Are they needed ?

Reply:
They are used for cinemachine. Not necessary for the project.